### ISSUE TITLE
Readme incomplete

### ISSUE THREAD

**Issue by mmadson** [TeamTopologies/Team-Cognitive-Load-Assessment#1]
Hi there, the `How To Use` section of the README appears to be incomplete / missing; would love to find some additional insight on how to use the example google form. Assume it may be explained in the book?

**Comment by manupaisable** [comment]
Yes, we need to add some more explanation to the README. The form is an example, as you say. There are questions on building, testing, releasing, and so on, that are fairly common to most software teams. But you need to think about questions specific to your organization, technology, and business domain, in order to highlight potential issues around too much or too little cognitive load.
